### PR TITLE
Chat UI

### PR DESCRIPTION
**Pull request by Miminiverse**
The following Pull request includes:

Create .d.ts files for image and gif files
Update the emoji in input field in Chat Page
Add in a Welcome page with welcome message + add the Bird gif in assets folder

This update is in reference to the following Jiras:
https://bbpracticum.atlassian.net/browse/CP2-20?atlOrigin=eyJpIjoiZjc5YjFiNDk5NTY3NDg1OWIxOTlmYWEyODQ5YjEwNWMiLCJwIjoiaiJ9

### CODE REVIEW

File: src/assests/Bird.gif
Comment (royemosby) [inline src/assests/Bird.gif:1]: 😍

**Review by royemosby — COMMENTED** [review]
👍

**Review by Said-laasri — APPROVED** [review]
**Great Job @Miminiverse :+1:** 

your bird gif is displaying at the top of the chat :1st_place_medal: 
Your emoji are displayed in the input after selecting one  :100: 

Comments:
the space between the emoji container and the input chat needs to be reduced :1234: 

Great job again  :+1: 


**Happy Coding**

**Review by royemosby — APPROVED** [review]
GTG

**Pull request merged**